src/CustardCommands/CreateLeafCommand.php

```diff
@@ -9,54 +9,61 @@
 
 class CreateLeafCommand extends CustardCommand
 {
-    protected function configure()
-    {
-        parent::configure();
-
-        $this->setName("leaf:create-leaf");
-        $this->addArgument("name", InputOption::VALUE_OPTIONAL, "The name of the leaf class to create.");
-    }
-
-    private function getNamespaceFromPath()
-    {
-        $map = include(VENDOR_DIR."/composer/autoload_psr4.php");
-        $path = getcwd();
-
-        foreach($map as $stubNamespace => $stubPaths ){
-            foreach($stubPaths as $stubPath) {
-                if (stripos($path, $stubPath) === 0) {
-                    // Found the right stub.
-                    $folders = str_replace($stubPath, "", $path);
-                    $namespace = rtrim($stubNamespace . trim(str_replace("/", '\\', $folders), "\\"), "\\");
-
-                    return $namespace;
-                }
-            }
-        }
-
-        return false;
-    }
-
-    protected function execute(InputInterface $input, OutputInterface $output)
-    {
-        $name = $input->getArgument("name");
-
-        if (sizeof($name) == 0) {
-            $name = $this->askQuestion("Enter the name for the Leaf class", "", true);
-        } else {
-            $name = $name[0];
-        }
-
-        $namespace = $this->getNamespaceFromPath();
-        $namespaceStatement = "";
-
-        if ($namespace){
-            $namespaceStatement = "
+	protected function configure()
+	{
+		parent::configure();
+
+		$this->setName("leaf:create-leaf")
+			->addArgument("name", InputOption::VALUE_OPTIONAL, "The name of the leaf class to create.")
+			->addOption('viewbridge');
+	}
+
+	protected function getNamespaceFromPath()
+	{
+		$map = include(VENDOR_DIR."/composer/autoload_psr4.php");
+		$path = getcwd();
+
+		foreach($map as $stubNamespace => $stubPaths ){
+			foreach($stubPaths as $stubPath) {
+				if (stripos($path, $stubPath) === 0) {
+					// Found the right stub.
+					$folders = str_replace($stubPath, "", $path);
+					$namespace = rtrim($stubNamespace . trim(str_replace("/", '\\', $folders), "\\"), "\\");
+
+					return $namespace;
+				}
+			}
+		}
+
+		return false;
+	}
+
+	protected function execute(InputInterface $input, OutputInterface $output)
+	{
+		$name = $input->getArgument("name");
+		$generateViewbridge = $input->getOption('viewbridge') != null;
+
+		if (sizeof($name) == 0) {
+			$name = $this->askQuestion("Enter the name for the Leaf class", "", true);
+		} else {
+			$name = $name[0];
+		}
+
+		$viewBridgeMethods = $generateViewbridge ? $this->getViewBridgeMethods($name) : "";
+		$viewBridgeUseStatement = $generateViewbridge ? "
+        use Rhubarb\Leaf\Leaves\LeafDeploymentPackage;
```

Comment:
extra whitespace at the start of this line - might be untidy in the produced file